### DISCUSSION TITLE
Install starship without sudo privileges/write access to /usr/local/bin

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#3097] [Q&A] [answered]
Can the prompt be installed to the home directory of a user, if the user doesn't have permissions to write to `/usr/local/bin`?

**Comment by matchai** [comment, marked as answer]
Yes, you can define the installation directory with the `--bin-dir` flag, like so:
```sh
sh -c "$(curl -fsSL https://starship.rs/install.sh)" -- --bin-dir $HOME
```
